src/System/Terminal.hsc

```diff
@@ -4,6 +4,8 @@ module System.Terminal
 ( getTerminalWidth
 ) where
 
+#ifdef WINDOWS
```

Comment:
Looks like this should be #ifndef

Reply:
Thanks for catching that!  BTW I'm still iterating until I get something reliable on all the builds.  I'll squash all these commits when I'm ready and note that it's ready for real review.